src/Selenium2Driver.php

```diff
@@ -163,23 +179,13 @@ public function getDesiredCapabilities()
     /**
      * Sets the WebDriver instance
      *
-     * @param WebDriver $webDriver An instance of the WebDriver class
+     * @param RemoteWebDriver $webDriver An instance of the WebDriver class
      */
-    public function setWebDriver(WebDriver $webDriver)
+    public function setWebDriver(RemoteWebDriver $webDriver)
```

Comment:
does this setter even make sense ? RemoteWebDriver is an equivalent of the session in the instaclick library. The class set there previously is the factory for the session (which is not an object which exists in the facebook library as they use a static method instead)

Reply:
removed this method